src/components/NcAvatar/NcAvatar.vue

```diff
@@ -710,28 +707,38 @@ export default {
 
 	&--with-menu {
 		cursor: pointer;
-		:deep(.v-popper) {
+		.action-item {
 			position: absolute;
 			top: 0;
 			left: 0;
 		}
-		.icon-more {
+		:deep(.action-item__menutoggle) {
 			cursor: pointer;
 			opacity: 0;
 		}
 		&:focus,
-		&:hover {
-			.icon-more {
+		&:hover,
+		&#{&}-loading {
+			:deep(.action-item__menutoggle) {
 				opacity: 1;
 			}
 			img {
 				opacity: 0.3;
 			}
 		}
-		.icon-more,
+		:deep(.action-item__menutoggle),
 		img {
 			transition: opacity var(--animation-quick);
 		}
+		:deep() {
+			.button-vue,
+			.button-vue__icon {
+				height: var(--size);
+				min-height: var(--size);
+				width: var(--size) !important;
+				min-width: var(--size);
+			}
+		}
```

Comment:
Can we add a size/small prop to `NcButton`instead of deep internal classes overrides?